assets/queries/k8s/anonymous_auth_is_not_set_to_false/query.rego

```diff
@@ -0,0 +1,55 @@
+package Cx
+
+import data.generic.common as common_lib
+
+CxPolicy[result] {
+	resource := input.document[i]
+	container := resource.spec.containers[j]
+
+	inArray(container.command, "kube-apiserver")
+	not common_lib.valid_key(container, "args")
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": "spec.containers.args",
+		"issueType": "MissingAttribute",
+		"keyExpectedValue": sprintf("spec.containers[%d] should have 'args' attribute", [j]),
+		"keyActualValue": sprintf("spec.containers[%d] does not contains 'args' attribute", [j]),
+	}
+}
+
+CxPolicy[result] {
+	resource := input.document[i]
+	container := resource.spec.containers[_]
+
+	inArray(container.command, "kube-apiserver")
+	inArray(container.args, "--anonymous-auth=true")
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": "spec.containers.args",
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "--anonymous-auth flag should be set to false",
+		"keyActualValue": "--anonymous-auth flag is set to true",
+	}
+}
+
+CxPolicy[result] {
+	resource := input.document[i]
+	container := resource.spec.containers[_]
+
+	inArray(container.command, "kube-apiserver")
+	not inArray(container.args, "--anonymous-auth=false")
```

Comment:
Just a hint: Changing this to something like
```
containerArgs := object.get(container, "args", {})
not inArray(containerArgs, "--anonymous-auth=false")
```
would enable you to omit the other `MissingAttribute`. It would also ensure that only actionable info is shown (meaning that "should contain args" still doesn't hint what exactly should be specified as args)